src/test/java/com/google/firebase/auth/FirebaseTokenVerifierImplTest.java

```diff
@@ -443,6 +455,258 @@ public void testBuilderNoDocUrl() {
     fullyPopulatedBuilder().setDocUrl(null).build();
   }
 
+  @Test
```

Comment:
Might be just easier to create a new test class `EmulatorFirebaseTokenVerifierImpleTest` for all these new test cases. You can do the env set up in the `@Before` fixture.

Reply:
Done.